### ISSUE TITLE
Changing defaults for profiles (roads etc)? Possible?

### ISSUE THREAD

**Issue by ewebgh33** [vvoovv/blender-geo#553]
Hi @vvoovv 

Importing a bunch of tiles to make a full city, which is then going to be exported out to FBX.

Is there a way to change the default profile widths so I can make roads a little wider on import, without having to adjust manually for every import I do?

Thanks!

**Comment by ewebgh33** [comment]
I was hoping I could do this numerically, where roads have a certain width in meters or other units, and I can just edit the numbers and see roads  update.

From the docs:
"Perform scaling of the profle vertices or simply move them. The corresponding road or path will change interactively as you edit its profile."

Full disclosure, I come from a C4D background, and not so great at Blender.
As far as I can tell, these curves are not "lines" with a dimension to them (equivalent in C4D is a line object you set object properties to whatever dimensions/length you want)?
They're lines defined by a start and end point, and thus have no property "length" that can be input with numbers?

**Comment by vvoovv** [comment]
@ewebgh33 

Have found the road profiles after an import?

**Comment by ewebgh33** [comment]
Yes, but "after import" means if I need to make 20 tiles to reassemble the city, I need to adjust the profiles in 20 files and hope they're the same, since I can't seem to change them numerically, and instead have to scale them by eyeball. Is this correct?

Is there a way to scale them and somehow resave the default that Bl-OSM is pulling in when the import occurs, so that I can set and forget?

Thanks

**Comment by vvoovv** [comment]
All you tiles should share the same profile objects. Edit a profile object for one tile and it will be update for the other tiles.
It's also possible to edit the profile objects in a script.

**Comment by ewebgh33** [comment]
I'm not sure what you mean. Where is the file containing the profile objects, so I can edit them and have that apply to all future imports?

Can we also change the default base mesh resolution for all terrain imports?

**Comment by vvoovv** [comment]
Just keep those profile objects for subsequent imports. They will be reused by subsequent imports.

![image](https://github.com/user-attachments/assets/7c3ff9de-1fd9-42d9-8974-641761af0d5b)

**Comment by ewebgh33** [comment]
Then I need to import /append them every time I want to use blOSM. Is there no way to change the defaults??

Also how about the base resolution of the terrain. I want a less dense mesh more often than not.

**Comment by vvoovv** [comment]
Save the file with the profiles and use it for your imports.

Use the option `Ratio` on the terrain panel to reduce the terrain resolution.

**Comment by ewebgh33** [comment]
Ratio has preset values we can't change. So its 100% or 25% and down. Can't input 90%, 50%. Isn't the terrain a displacement coming from a heightmap, so we should be able to set polycount to anything really?

**Comment by vvoovv** [comment]
It's an array of points.

**Comment by ewebgh33** [comment]
I see. So ratio limitations are a result of what point arrays OSM supplies via API or blender/blOSM is doing something to reduce point count on application of the ratio setting? If you could kindly explain in more than a few words I would know how to proceed or what I am limited to working with in a different way. EG I have a remesh going because 100% is too high and 25% is too low but the remesh is also very slow. 
Another option is I bake out a height map in 8 or 16k then apply that as displacement to a plane where I can put any subdivisions I want.
But I am asking as those seem like workarounds and I had hoped there was a simpler way. OK if not, but that is what I am trying to understand.
